### PR TITLE
Update fdk-go landing page

### PR DESCRIPTION
**Pull request by michael-w-williams**
Updates fdk-go landing page to new template. Probably need to remove examples directory as that just has redundant information from what I can tell, but first things first. Let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-
-# Installing fdk-go
-
-```sh
-go get github.com/fnproject/fdk-go
-```
-
-or your favorite vendoring solution :)
+# Go FDK Documentation
+This is documentation for the Go function development kit (FDK). The Go FDK  provides convenience functions for writing Go Fn code.
```
Comment (shaunsmith) [inline README.md]: ...for the Go function development kit (FDK) which provides...

File: README.md [hunk 2]
```diff
@@ -1,25 +1,16 @@
...
 
-# Examples
+## User Information
```
Comment (shaunsmith) [inline README.md]: We do need a link to the top of the doc set for Fn, but one of these links are actually related to the Go FDK.  What we need is something on how to obtain the Go FDK.  There are two options: 1) it's in the build and runtime Go images, and 2) if someone were building a custom runtime image using a Dockerfile, how do they include it in their program dependencies?

File: README.md (hunk 1 above)
Comment (michael-w-williams) [reply README.md]: Updated the paragraph and pushed.

File: README.md (hunk 2 above)
Comment (michael-w-williams) [reply README.md]: Adding issue and doc updates. Will provide that info shortly.

Comment (michael-w-williams) [reply README.md]: Addressing the above by adding issues to fdk-go:

* Add https://fnprojec.io to repo description at top of the page. See: https://github.com/fnproject/fdk-go/issues/47
* Add new content on Go FDK development. See: ttps://github.com/fnproject/fdk-go/issues/48
* This PR also closes: https://github.com/fnproject/fdk-go/issues/40

**Comment by rdallman** [comment]
> Updates fdk-go landing page to new template. Probably need to remove examples directory as that just has redundant information from what I can tell, but first things first. Let me know what you think.

yea, i think we can remove it. there's a top level example that goes to godoc now too https://godoc.org/github.com/fnproject/fdk-go#example-package

**Pull request merged**